CMakeLists.txt

```diff
@@ -13,10 +12,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# =======================================================================
 #
 
-cmake_minimum_required(VERSION 3.12.0)
+cmake_minimum_required(VERSION 3.13...3.17)
```

Comment:
cmake_policy(VERSION <min>[...<max>]).  This syntax was introduced in CMake V12.4.

I've set the min policy to 3.13, although I think 3.15 is more suitable.